plugin/codeium.vim

```diff
@@ -60,7 +60,10 @@ if !get(g:, 'codeium_disable_bindings')
 endif
 
 call s:SetStyle()
-call timer_start(0, function('codeium#server#Start'))
+
+if g:codeium_enabled
```

Comment:
When I run this I get the following error:

```
Error detected while processing /home/rahul/.config/nvim/pack/Exafunction/start/codeium.vim/plugin/codeium.vim:
line   64:
E121: Undefined variable: g:codeium_enabled
Press ENTER or type command to continue
```

I think you should replace this line with

```
if codeium#Enabled()
```

Reply:
done